obsreport/obsreport_exporter.go

```diff
@@ -110,7 +110,11 @@ func (exp *Exporter) recordMetrics(ctx context.Context, numSent, numFailedToSend
 		return
 	}
 	// Ignore the error for now. This should not happen.
-	_ = stats.RecordWithTags(ctx, exp.mutators, sentMeasure.M(numSent), failedToSendMeasure.M(numFailedToSend))
+	if numFailedToSend > 0 {
+		_ = stats.RecordWithTags(ctx, exp.mutators, sentMeasure.M(numSent), failedToSendMeasure.M(numFailedToSend))
```

Comment:
I think this will work, but I wonder whether it would be better to change the `errorNumberView` definition to use `view.Sum()` and here observe `0` or `1`.  That seems to me a more direct expression of the intent.

Reply:
The whole idea of count is to avoid multiple recordings.